types/sinon/index.d.ts

```diff
@@ -14,6 +14,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
+import * as FakeTimers from 'sinonjs__fake-timers';
```

Comment:
see if `@sinonjs/fake-timers` works here, it should be resolved right from what i remember

Reply:
nope, getting errors.

using `@sinonjs/fake-timers` gives the error:
```
ERROR: 1:29  expect  TypeScript@next compile error: 
Cannot find module '@sinonjs/fake-timers' or its corresponding type declarations.
```

using `@types/sinonjs__fake-timers` gives the errors:
```
ERROR: 1:29  expect  TypeScript@next compile error: 
Cannot find module '@types/sinonjs__fake-timers' or its corresponding type declarations.
ERROR: 1:29  expect  TypeScript@next compile error: 
Cannot import type declaration files. Consider importing 'sinonjs__fake-timers' instead of '@types/sinonjs__fake-timers'.
```

Reply:
it should be `@sinonjs/fake-timers` for sure so im not sure whats up with that... ill have a poke around

Reply:
i have this working:

1. need to specify the path mapping in tsconfig for sinon
2. need to declare `@sinonjs/fake-timers` as an ambient module (i think)